tests/components/renault/test_sensor.py

```diff
@@ -253,9 +253,9 @@ async def test_sensor_throttling_after_init(
 @pytest.mark.parametrize(
     ("vehicle_type", "vehicle_count", "scan_interval"),
     [
-        ("zoe_40", 1, 300),  # 5 coordinators => 5 minutes interval
```

Comment:
zoe_40 was updated from 5 to 4 coordinators... so using zoe_50 instead to show difference from captur_fuel